_faq/cep-and-its-importance.md

```diff
@@ -0,0 +1,25 @@
+---
+question: "What is a cEP and why it is so important to do analysis pre-gsoc and
+ design as early as possible, to be finalized in the cEP?"
+---
+A **cEP (coala Enhancement Proposal)** is proposed to the coala community to
+ discuss the new major features that the contributor intends to implement during
```

Comment:
please remove the unnecessary whitespaces at the start of line 6, 7, 15, 16, 19, 20, 21, 22

Reply:
Actually, when I separated a long line to multiple lines, that whitespace at the starting of each line came up when I pressed enter key after a certain a word and that whitespace at beginning of sentence was actually the space between any two words, say between `to` and `discuss`. I thought that space is necessary at the beginning so that content can be rendered properly on webpage but I was wrong, it is still getting rendered properly even when the extra beginning whitespaces's are removed. 😃

Reply:
> Actually, when I separated a long line to multiple lines, that whitespace at the starting of each line came up

There might be something wrong with your editor setting because in my case a newline doesn't add a whitespace to the beginning of the line. Figure it out :wink: 

>  I thought that space is necessary at the beginning so that content can be rendered properly on webpage but I was wrong

I am not sure whether whitespace would have anything to do with the proper display of content, but if you are unsure about that just build and run the site locally without the whitespace  and see if it is being rendered correctly.

Reply:
I checked it out by running the websute locally, and the content is getting rendered correctly as it should be.

Reply:
Any more changes need to be made ?

Reply:
None from my side let others review it.

Reply:
Can you approve the changes ?